features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2

```diff
@@ -767,7 +767,6 @@
         </BlockCondition>
         <JMSConnectionDetails>
             <Enabled>{{apim.throttling.enable_decision_connection}}</Enabled>
-            <Destination>{{apim.throttling.jms.destination}}</Destination>
```

Comment:
We can remove default config value 'apim.throttling.jms.destination' from here[1]

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json#L88

Reply:
Fixed in https://github.com/bhathiya/carbon-apimgt/commit/725c37293a88cf5d574c6eaedae36fd906fef75f